components/application-service/base/kustomization.yaml

```
@@ -12,4 +12,19 @@ images:
 namespace: application-service
 
 patches:
-  - ./manager_resources_patch.yaml
+- ./manager_resources_patch.yaml
+- patch: |-
+    - op: add
+      path: /metadata/labels
+      value:
+        appstudio.redhat.com/namespacetype: "controller"
+    - op: add
+      path: /metadata/annotations
+      value:
+        k8s.ovn.org/acl-logging: |-
+      {
+        "deny": "info",
+        "allow": "info"
+      }
```

the same here, plus I don't think that this format of the annotation value is supported by kustomize
```suggestion
    - op: add
      path: /metadata/annotations/k8s.ovn.org~1acl-logging
      value:
        '{"deny": "info", "allow": "info"}'
```